paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -70,12 +70,12 @@ public abstract class PrepareResourcesTask : DefaultTask() {
 
     val mainPackage = packageName.get()
     val resourcePackageNames = if (nonTransitiveRClassEnabled.get()) {
-      buildList {
+      buildSet {
         add(mainPackage)
         artifactFiles.files.forEach { file ->
           add(file.useLines { lines -> lines.first() })
         }
-      }
+      }.toList()
```

Comment:
nit: maybe this `resourcePackageNames` can just be a set instead of a list to remove the extra type transform.

Reply:
The only thing is that I would have to change `Config` to accept a `Set` instead of a `List`. And maybe for consistency it would require all the lists in `Config` to be sets as well. Let me know if you folks are ok with that.

I could also just use `distinct()` here and avoid the creation of a `Set`. Let me know what you think it's best.

Reply:
I think it should be a list, order matters, or not? probably should be `buildList { }.distinct()` because `buildSet {}.toList()` (semantically) can have arbitrary order.

Reply:
Changed to `distinct`